_posts/2015-06-16-hard-fork-policy.md

```diff
@@ -8,31 +8,15 @@ title: "Bitcoin.org Hard Fork Policy"
 permalink: /en/posts/hard-fork-policy.html
 date: 2015-06-16
 ---
-Contentious hard forks are bad for Bitcoin. At the very best, a
-contentious hard fork will leave people who chose the losing side of the
-fork feeling disenfranchised. At the very worst, it will make bitcoins
-permanently lose their value. In between are many possible outcomes, but
-none of them are good.
-
-The danger of a contentious hard fork is potentially so significant
-that Bitcoin.org has decided to adopt a new policy:
-
-> Bitcoin.org will not promote software or services that will leave the
-> previous consensus because of an intentional and contentious hard fork attempt.
-
 This policy applies to full node software, such as Bitcoin Core,
-software forks of Bitcoin Core, and alternative full node
-implementations.
+software forks of Bitcoin Core, and alternative full node implementations.
 
 It also applies to wallets and services that have the ability to detect
-the contentious hard fork, and which release code or make announcements
-indicating that they will cease operating on the side of the previous
-consensus.
-It does not apply to software that cannot detect the contentious hard
-fork and which continues doing whatever it would've done anyway.
+hard forks, and which release code or make announcements indicating that
+they will cease operating on the side of the majority consensus.
+
+>>Wallet authors and service providers who would like to discuss their opinions
+>>or contributions relating to hard forks should open an issue or submit
+>>a pull request specific to their requested change.
```

Comment:
Question purely about phrasing: do I understand correctly that you're suggesting that if Alice doesn't like Bob's wallet because it does something related to a hard fork, she should open a PR removing Bob's wallet from the site?